tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/OpenAIService.cs

```diff
@@ -0,0 +1,61 @@
+using System.Text.Json;
+using Azure.AI.OpenAI;
+using Azure.Core;
+using Azure.Identity;
+using Azure.Sdk.Tools.Cli.Models;
+using OpenAI.Chat;
+
+namespace Azure.Sdk.Tools.Cli.Services;
+
+/// <summary>
+/// Interface for Azure OpenAI client to enable dependency injection and testing
+/// </summary>
+public interface IAzureOpenAIClient
+{
+    /// <summary>
+    /// Gets a chat client for the specified model deployment
+    /// </summary>
+    /// <param name="model">The model deployment name</param>
+    /// <returns>A chat client instance</returns>
+    ChatClient GetChatClient(string model);
+}
+
+
+/// <summary>
+/// Wrapper implementation of IAzureOpenAIClient that delegates to the actual AzureOpenAIClient
+/// </summary>
+public class AzureOpenAIClientService : IAzureOpenAIClient
```

Comment:
By convention, all C# types should have their own file.   You don't want to piggy-back this way.

Reply:
I'll change it, but you're making me the odd-one-out in this folder. :)

Reply:
I am not avoiding this responsibility, but I ended up not needing the file so I deleted it.